tests/protocol/zone/test.js

```diff
@@ -0,0 +1,95 @@
+if (typeof PATH_TO_THE_REPO_PATH_UTILS_FILE === 'undefined') PATH_TO_THE_REPO_PATH_UTILS_FILE = "https://raw.githubusercontent.com/highfidelity/hifi_tests/master/tests/utils/branchUtils.js";
+Script.include(PATH_TO_THE_REPO_PATH_UTILS_FILE);
+var autoTester = createAutoTester(Script.resolvePath("."));
+
+autoTester.perform("Protocol sanity", Script.resolvePath("."), "secondary", function(testType) {
+    Script.include('../common.js');
+    setup();
+    
+    var object;
+    
+    var setProperties = {
+        lifetime: LIFETIME,
+        description: "A55ADEiOjHg6F",
+        type: "Zone",
+        name: "Kih85Rfg",
```

Comment:
Sorry, I should have been clearer.  Why are we using these random strings for the description and name?

Reply:
This is on purpose, as all we are checking is that the string is not being shifted due to a protocol error.